app/src/main/AndroidManifest.xml

```diff
@@ -73,21 +76,21 @@
         <activity
             android:name=".activity.LogicalAnalyzerActivity"
             android:configChanges="screenSize|orientation"
-            android:screenOrientation="landscape" />
+            android:screenOrientation="sensorLandscape" />
         <activity android:name=".activity.SensorDataLoggerActivity" />
         <activity
             android:name=".activity.MultimeterActivity"
-            android:screenOrientation="portrait" />
+            android:screenOrientation="user" />
         <activity
             android:name=".activity.PowerSourceActivity"
-            android:screenOrientation="portrait"
+            android:screenOrientation="user"
             android:windowSoftInputMode="stateHidden|adjustResize" />
         <activity
             android:name=".activity.LuxMeterActivity"
             android:screenOrientation="portrait" />
         <activity
             android:name=".activity.WaveGeneratorActivity"
-            android:screenOrientation="portrait" />
+            android:screenOrientation="user" />
```

Comment:
Revert back and use a layout file for tabs. Refer to comments for a screenshot.

Reply:
Ok sir , will do that